include/measurement_kit/net/buffer.hpp

```diff
@@ -93,52 +47,48 @@ class Buffer : public NonCopyable, public NonMovable {
      */
 
     Buffer &operator<<(evbuffer *source) {
-        if (source == NULL)
-            throw std::runtime_error("source is NULL");
+        if (source == nullptr) throw std::runtime_error("source is nullptr");
         if (evbuffer_add_buffer(evbuf, source) != 0)
             throw std::runtime_error("evbuffer_add_buffer failed");
-        return (*this);
+        return *this;
```

Comment:
Just to understand, is this just a stylistic change or does it change the meaning?